cmd/sonobuoy/app/worker.go

```diff
@@ -35,6 +37,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	pathResultsByNode = "/api/v1/results/by-node"
+	pathResultsGlobal = "/api/v1/results/global"
```

Comment:
I could have tried to export these from the aggregator's package, but the value that those really need has a parameterized suffix (e.g. `/api/v1/results/by-node/{node}/{type}`) and it felt a bit forced to add the new values, export them, and use them to build the existing values by adding back the suffix.

At the time I thought this was a bit overkill, but and it would be normal for a client to reference the paths of things like that.

Update pending...